src/index.html

```diff
@@ -449,9 +449,10 @@ <h2>Help</h2>
                                     <li class="accordion-item is-active" data-accordion-item>
                                         <a class="accordion-title"><h5>What is the European Variation Archive (EVA)?</h5></a>
                                         <div class="accordion-content" data-tab-content>
-                                            <p>The European Variation Archive (EVA) is EMBL-EBI's open-access genetic variation archive. The EVA accepts submission of <b>all</b> types of <b>precise</b> genetic variants, ranging from single nucleotide polymorphisms to large structural variants, observed in germline or somatic sources, from any organism. The EVA permits access to these data at two distinct levels:</p>
-                                            <p> i) The raw variant data as was submitted to the EVA, via the <a href="http://www.ebi.ac.uk/eva/?Study%20Browser&browserType=sgv" target="_blank">EVA Study Browser</a></p>
-                                            <p> ii) The normalised and processed variant data, via the <a href="http://www.ebi.ac.uk/eva/?Variant%20Browser" target="_blank">EVA Variant Browser</a> and <a href="http://www.ebi.ac.uk/eva/?API" target="_blank">EVA API</a></p>
+                                            <p>The European Variation Archive (EVA) is a primary deposition archive for genetic variation. The EVA is part of the EMBL-EBI infrastructure and accepts submission of <b>all</b> types of <b>precise</b> genetic variants, ranging from single nucleotide polymorphisms to large structural variants, observed in germline or somatic sources, from any organism. The EVA provide access to these data at multiple levels:</p>
+                                            <p> i) The original variation data in VCF format as was submitted to the EVA, via the <a href="http://www.ebi.ac.uk/eva/?Study%20Browser&browserType=sgv" target="_blank">EVA Study Browser</a></p>
+                                            <p> ii) Each individual variant and their genotypes, via the <a href="http://www.ebi.ac.uk/eva/?Variant%20Browser" target="_blank">EVA Variant Browser</a> and <a href="http://www.ebi.ac.uk/eva/?API" target="_blank">EVA API</a></p>
+                                            <p> ii) All variant associated with RS ids found in one assembly, via the <a href="https://www.ebi.ac.uk/eva/?RS-Release" target="_blank">EVA RS release</a></p>

```

Comment:
```suggestion
                                            <p> iii) All variant loci associated with RS ids found in one assembly, via the <a href="https://www.ebi.ac.uk/eva/?RS-Release" target="_blank">EVA RS release</a></p>

```
(Or just `All variants...`)